odoo/exceptions.py

```diff
@@ -107,3 +114,53 @@ def __init__(self, msg, tb):
 
 class QWebException(Exception):
     pass
+
+
+class ViewError(except_orm, ValueError):
```

Comment:
I would remove the inheritance with `ValueError`.
I don't think it brings anything useful.

Reply:
there may be places that try to render a view and check for a `ValueError`, I did it since the task is supposed to be somewhat backportable

Reply:
lets keep the changes minimal dont add a new type of exception